service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/PlacementController.java

```diff
@@ -81,12 +88,18 @@ public AboutToStartOrSubmitCallbackResponse handleAboutToSubmit(@RequestBody Cal
         UUID childId = getSelectedChildId(caseDetails, caseData);
         Element<Child> child = placementService.getChild(caseData, childId);
 
-        Placement placement = mapper.convertValue(caseDetails.getData().get("placement"), Placement.class)
+        Placement currentPlacement = mapper.convertValue(caseDetails.getData().get("placement"), Placement.class)
             .setChild(child);
 
-        caseProperties.put("placements", placementService.setPlacement(caseData, placement));
+        Placement previousPlacement = placementService.getPlacement(caseData, child);
```

Comment:
should we prefer `callbackRequest.getCaseDetailsBefore()` for getting previous `CaseData->Placement` to `placementService.getPlacement(caseData, child)`?

Reply:
I'd considered this but it gives little benefit in my opinion. At the point of call caseDetailsBefore and caseDetails will return the same data.